dspace-api/src/main/java/org/dspace/app/util/GoogleBitstreamComparator.java

```diff
@@ -86,8 +86,10 @@ public int compare(Bitstream b1, Bitstream b2) {
         if (priority1 > priority2) {
             return 1;
         } else if (priority1 == priority2) {
-            if (b1.getSizeBytes() <= b2.getSizeBytes()) {
+            if (b1.getSizeBytes() < b2.getSizeBytes()) {
                 return 1;
```

Comment:
Isn't this backward?  Shouldn't `compare(b1, b2)` return -1 if b1 is smaller than b2, and 1 if b1 is larger than b2, priority being equal?

Reply:
Correct, I didn't change the logic here...it was already sorting largest first.  I just split up this `if` into an `if` and and `else if`